docs/software-platforms/aws/working-with-snapshots.md

```diff
@@ -0,0 +1,114 @@
+
+1. Navigate to the `Snapshots` section of the LSfR console.
+Here you will see a list of all your virtual computers, disks, and any existing snapshots.
+
+    <figure markdown="span">
+        ![Snapshot menu](../../img/snapshots-1.png){width="600"}
+    </figure>
+
+1. Choose the virtual computer or disk that you would like to snapshot from the list and select `Create snapshot`.
+
+    <figure markdown="span">
+        ![Virtual computer snapshot](../../img/snapshots-2.png){width="600"}
+    </figure>
+
+1. You will then be prompted to provide a descriptive name for your snapshot.
+It can be helpful to include the date of the snapshot in the name.
+Once a name has been created, click `Create snapshot`.
+
+    <figure markdown="span">
+        ![Create snapshot](../../img/snapshots-3.png){width="600"}
+    </figure>
+
+1. Each virtual computer and disk will have a drop-down menu listing all snapshots associated with that virtual computer or disk.
+Once your snapshot has been created you should be able to see the snapshot listed in that menu.
+
+    <figure markdown="span">
+        ![View snapshot](../../img/snapshots-4.png){width="600"}
+    </figure>
+
+
+## Creating a new virtual computer or disk from a snapshot
+
+1. Find the previous virtual computer or disk that contains the snapshot you would like to use for creating a new virtual computer or disk.
+Open up the snapshot drop-down menu and select the snapshot you would like to use.
+
+1. With the snapshot selected, use the `Actions` drop-down menu to select `Create virtual computer`.
+If creating a disk from a disk snapshot, select `Create disk`.
+
+    <figure markdown="span">
+        ![Restore snapshot](../../img/snapshots-5.png){width="600"}
+    </figure>
+
+1. Give your new virtual computer or disk a descriptive name.
+Pick the instance size or disk size from the menu options.
+
+    By default, the same instance size or disk size will be selected, but if you would like to re-size please choose a different option from the menu.
+
+    - Reference these [guidelines when choosing an instance size](./creating-vcs.md#choosing-an-instance).
+    - Remember that you will have a [monthly budget for any computational resources you use](../../getting-started/accessing-resources/getting-access-to-compute.md#monthly-budget).
+
+    <figure markdown="span">
+        ![Select size](../../img/snapshots-6.png){width="700"}
+    </figure>
+
+1. Once you have chosen your size, select either `Create virtual computer` or `Create disk`.
+Your new virtual computer will be listed under the `Virtual computers` section and new disks can be found in the `Storage` section of LSfR.
+
+    If you are creating a new virtual computer, please be sure to follow the instructions below to [create a cost control rule](#creating-a-cost-control-rule).
+    A cost control rule ensures that your virtual computer does not continue to run when not actively in use, lowering your total cost.
```

Comment:
I wonder if we should put this in a warning admonition? We don't want them to gloss over this